packages/frontend/web/components/Footer.tsx

```diff
@@ -110,33 +190,58 @@ const FooterLinks: React.FC<{
         return <ul key={key}>{linkList}</ul>;
     });
 
-    return <div className={footerList}>{linkGroups}</div>;
+    const rrLinks = (
+        <div className={readerRevenueLinks}>
+            <ReaderRevenueLinks
+                urls={nav.readerRevenueLinks.footer}
+                edition={edition}
+                dataLinkNamePrefix={'footer : '}
+                noResponsive={true}
+            />
+        </div>
+    );
+
+    return (
+        <div className={footerList}>
+            {linkGroups}
+            {rrLinks}
+        </div>
+    );
 };
 
 const year = new Date().getFullYear();
 
 export const Footer: React.FC<{
+    nav: NavType;
+    edition: Edition;
     pageFooter: FooterType;
-}> = ({ pageFooter }) => (
+}> = ({ nav, edition, pageFooter }) => (
     <footer className={footer}>
         <Container className={footerInner}>
-            <iframe
-                title="Guardian Email Sign-up Form"
-                src="https://www.theguardian.com/email/form/footer/today-uk"
-                scrolling="no"
-                seamless={true}
-                id="footer__email-form"
-                className={emailSignup}
-                data-form-success-desc="We will send you our picks of the most important headlines tomorrow morning."
-                data-node-uid="2"
-                height="100px"
-                frameBorder="0"
-            />
-            <FooterLinks pageFooter={pageFooter} />
-            <div className={copyright}>
-                © {year} Guardian News & Media Limited or its affiliated
-                companies. All rights reserved.
+            <div className={footerItemContainers}>
+                <iframe
+                    title="Guardian Email Sign-up Form"
+                    src="https://www.theguardian.com/email/form/footer/today-uk"
+                    scrolling="no"
+                    seamless={true}
+                    id="footer__email-form"
+                    className={emailSignup}
+                    data-form-success-desc="We will send you our picks of the most important headlines tomorrow morning."
+                    data-node-uid="2"
+                    height="100px"
+                    frameBorder="0"
+                />
+
+                <FooterLinks
+                    nav={nav}
+                    edition={edition}
+                    pageFooter={pageFooter}
+                />
             </div>
         </Container>
+        <div className={copyright}>
```

Comment:
Why did this come out of the container? Seems we needed to add a bunch more CSS to handle it? I think we can move the border to the itemContainers

Reply:
I am not sure - think this predates my work. But will have a look.